v2/nodejs/jwt/createJWT.mdx

```diff
@@ -0,0 +1,26 @@
+---
+id: createJWT
+title: createJWT
+hide_title: true
+---
+
+## `createJWT(payload, validity)`
+
+### Parameters
+- `payload`
```

Comment:
Shouldn't payload be optional?

Reply:
So in the interface it isnt optional but in hindsight it makes to allow people to create JWT without passing custom payloads. Changed it here (as well as the function override docs) and made changes in the NodeJS PR